modules/light-clients/07-tendermint/types/update_test.go

```diff
@@ -100,6 +105,70 @@ func (suite *TendermintTestSuite) TestCheckHeaderAndUpdateState() {
 			},
 			expPass: true,
 		},
+		{
+			name: "successful update with identical header to a previous update",
+			setup: func(suite *TendermintTestSuite) {
+				clientState = types.NewClientState(chainID, types.DefaultTrustLevel, trustingPeriod, ubdPeriod, maxClockDrift, heightPlus1, commitmenttypes.GetSDKSpecs(), upgradePath, false, false)
```

Comment:
It worries me a little to use so much manual setup as opposed to just running `SetupClients`, but I guess updating these tests can be done another day